modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/ratings.js

```diff
@@ -489,9 +489,49 @@ AUI.add(
 
 						var thumbScore = instance._getThumbScores(json.totalEntries, json.totalScore);
 
+						instance._updateTitles(thumbScore);
 						instance._updateScores(thumbScore);
 					},
 
+					_updateTitles: function(thumbScore) {
+						var instance = this;
+
+						var nodes = instance.ratings.get('elements');
```

Comment:
Should we use `get` or `all`? Should the variable name be `nodes` or `elements`?